how-to-guides/validator-node.md

```diff
@@ -234,6 +202,60 @@ You should now be able to see your validator from
 After starting your node, please submit your node as a seed and peer to the
 [networks repository](https://github.com/celestiaorg/networks).
 
+## Optional: Delegate more stake to a validator
```

Comment:
Thank you for the PR @gluckzhang. why does this heading change from "### Delegate stake to a validator" to "## Optional: Delegate more stake to a validator" i.e. why is it "More" now?

Reply:
Hi @jcstein, thanks for reviewing it :) Because I moved the section later and I think it should be at the same level as another optional step. Maybe we can group some of the sub-sections if you think the page contains too many `##` headlines. Just let me know what you think and I will adjust it!

Here are the screenshots that show the difference.

Current proc version:
<img width="312" alt="image" src="https://github.com/user-attachments/assets/160d5119-ff48-47df-b632-d011aafada24" />

This PR:
<img width="293" alt="image" src="https://github.com/user-attachments/assets/0cbd31f7-2778-45be-97c5-b9547a481e00" />

Reply:
Oh sorry I just got your question. I agree `more` is not necessary here. Let me change it back to the original title :)